components/Navigation/NavigationContainer/index.stories.tsx

```diff
@@ -0,0 +1,30 @@
+import { useState } from 'react';
+import NavigationContainer from './';
+import type { Meta as MetaObj, StoryObj } from '@storybook/react';
+import type { FC } from 'react';
+
+type Meta = MetaObj<typeof NavigationContainer>;
+
+const Children: FC = () => (
+  <ul>
+    <li>Navigation item 1</li>
+    <li>Navigation item 2</li>
+    <li>Navigation item 3</li>
+  </ul>
+);
+
+export const Default = () => {
```

Comment:
For being consistent...
Don't forget to declare this above:
```js
type Story = StoryObj<typeof NavigationContainer>;
```
```suggestion
export const Default = (): Story => {
```